### PR TITLE
Changes to Views

### PR DESCRIPTION
**Pull request by litopia**
- Added Genes views, including: Genes list, gene list detail, edit gene list detail, and create gene list page
- Added Variables views, including: variables list, variables list detail, edit/create variables
- Added Projects views, including: projects list, project detail and upload project 
- Modified Cohorts existing UI
- Adapted navigations and dashboard to be consistent with overall color theme

### CODE REVIEW

File: accounts/views.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 COUNTDOWN_SECONDS = login_expiration_seconds + (60 * 15)
 
 logger = logging.getLogger(__name__)
-ACL_GOOGLE_GROUP = settings.OPEN_ACL_GOOGLE_GROUP
+# ACL_GOOGLE_GROUP = settings.OPEN_ACL_GOOGLE_GROUP
```
Comment (LoveAndCoding) [inline accounts/views.py:46]: Was this change needed for some particular reason or maybe just a merge issue?

Comment (litopia) [inline accounts/views.py:46]: I assume this is a merge issue. Or I probably comment it out because it throws errors on starting the application.
